Source/CDProject/GameMode/RoundGameMode.cpp

```
@@ -0,0 +1,244 @@
+void ARoundGameMode::Tick(float DeltaSeconds)
+{
+	Super::Tick(DeltaSeconds);
+	
+	if (_curMatchState==ECurMatchState::EMS_Waiting)
+	{
+		Countdown=FMath::CeilToInt(WaitingStartTime + WarmUpTime-GetWorld()->GetTimeSeconds());
+		//UE_LOG(LogGameMode, Log, TEXT("Countdown %f"), Countdown);
+		if (Countdown==-1)
+		{
+			SetCurMatchState(ECurMatchState::EMS_InGame);
+		}
```

Cppcheck (2.10-2)

[error] There is an unknown macro here somewhere. Configuration is required. If UPROPERTY is a macro then please configure it.

(unknownMacro)